docs/hub/main.md

```diff
@@ -94,6 +94,21 @@ Here is a video to learn more about it from our [course](http://hf.co/course)!
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/rkCly_cbMBk" title="Managing a repo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## How can I rename or transfer a repo?
+
+You can go to the settings page in any repo to achieve this. Note that there are certain limitations in terms of use cases.
+
+Moving can be used for three use cases ✅ 
+- Renaming a repository within same user.
+- Renaming a repository within same organization.
+- Transfering repository from user to an organization.
```

Comment:
```suggestion
- Transfering repository from user to an organization. The user must be part of the organization and have "write" or "admin" rights in the organization.
```
or something similar? :) cc @SBrandeis

Reply:
Note that this also holds for renaming a repo inside the same organization